### ISSUE TITLE
Eyedropper is not working

### ISSUE THREAD

**Issue by Linzerella** [kepi/chromeEyeDropper#258]
### Describe the bug

When I go to use the eyedropper from a webpage nothing happens - I redownloaded the app and then got this error message "no active palette... This is embarassing and should never happen, please report the bug." 

### Expected Behavior

Normally, the eyedropper works and the colors change as you move the cursor over them, this did not happen.

### Operating system

None

**Comment by Linzerella** [comment]
I fixed the issue by rebooting - my apologies!

**Comment by kepi** [comment]
@Linzerella thanks for letting me know. Btw. that really shouldn't happen. Did you youse ED before and it stopped working afer update or it was the new installation?

Thanks

**Comment by Linzerella** [comment]
The old installation always worked, and I did end up getting the new installation to work as well. Honestly , I think it was an issue on my end because half way through the day I realized another extension of mine didn’t work. When I restarted my computer it reset everything and now my extensions, including the latest version of eye dropper - all work.Sorry for the false alarm but thanks for the follow up!!!!On Jul 15, 2024, at 9:05 AM, Kepi ***@***.***> wrote:﻿
@Linzerella thanks for letting me know. Btw. that really shouldn't happen. Did you youse ED before and it stopped working afer update or it was the new installation?
Thanks
